opentracing/__init__.py

```diff
@@ -28,8 +28,10 @@
 from .propagation import InvalidCarrierException  # noqa
 from .propagation import SpanContextCorruptedException  # noqa
 from .propagation import UnsupportedFormatException  # noqa
+from .globaltracer import init_global_tracer  # noqa
+from .globaltracer import get_global_tracer  # noqa
 
 # Global variable that should be initialized to an instance of real tracer.
 # Note: it should be accessed via 'opentracing.tracer', not via
 # 'from opentracing import tracer', the latter seems to take a copy.
-tracer = Tracer()
+tracer = get_global_tracer()
```

Comment:
I got the reasoning here, and I agree that having this constraint is not ideal. In general, would be great that users don't have to think when it's the right time for the import.